experiment/nursery/create_bazel_cache_rcs.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# THIS IS TEMPORARY (!)
+# Once this moves out of experiment/ we will create these files in the bazel
+# images instead (!)
+# TODO(bentheelder): verify that this works and move it into the images
+CACHE_HOST="http://bazel-cache:8080"
```

Comment:
what's running here? is there an associated PR or tracking issue?

Reply:
oh, I read the PRs out-of-order.

Reply:
yeah, this is a follow up to https://github.com/kubernetes/test-infra/pull/6660